tests/behat/behat_mod_coursework.php

```diff
@@ -3657,4 +3657,28 @@ public function i_set_the_field_to_replacing_line_breaks($field, $value) {
         $value = str_replace('\n', chr(10), $value);
         $this->execute([behat_forms::class, 'i_set_the_field_to'], [$field, $value]);
     }
+
```

Comment:
I can't see how this is different from the core step:

When I click on "Delete version" "link" in the "H5P.FontIcons" "table_row"